manifests/10_cluster-operator.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: config.openshift.io/v1
+kind: ClusterOperator
+metadata:
+  name: service-catalog-controller-manager
+spec: {}
```

Comment:
can you provide a pointer to when you report available=true?

Reply:
In starter.go we create a ClusterOperatorStatus controller and then run it here: 
 https://github.com/openshift/cluster-svcat-controller-manager-operator/blob/9261f420a3db9556606c8ee0980a5e02a8f28d89/pkg/operator/starter.go#L99

we immediately get an event that triggers sync and we update the status:  https://github.com/openshift/library-go/blob/23ffddb2eeab32431fc585be3012b54616bf10b6/pkg/operator/status/status_controller.go#L152 showing true/false/false